portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Becca's Portfolio</h1>
+        <p>Welcome!</p>
+        <p>I'm a rising junior computer science major at Vanderbilt University</p>
```

Comment:
Nit: These don't need to be indented under the header.

Reply:
fixed